Code/CrySystem/GameWindow.cpp

```diff
@@ -197,12 +212,23 @@ static LRESULT CALLBACK WindowProc(HWND window, UINT msg, WPARAM wParam, LPARAM
 		}
 		case EVENT_SYSTEM_MENUPOPUPSTART:
 		{
-			gEnv->pSystem->GetISystemEventDispatcher()->OnSystemEvent(ESYSTEM_EVENT_CHANGE_FOCUS, 0, 0);
+			//gEnv->pSystem->GetISystemEventDispatcher()->OnSystemEvent(ESYSTEM_EVENT_CHANGE_FOCUS, 0, 0);
+
+			if (g_pGame)
+			{
+				g_pGame->ConfineCursor(false);
+			}
+
 			break;
 		}
 		case EVENT_SYSTEM_MENUPOPUPEND:
 		{
-			gEnv->pSystem->GetISystemEventDispatcher()->OnSystemEvent(ESYSTEM_EVENT_CHANGE_FOCUS, 1, 0);
+			//gEnv->pSystem->GetISystemEventDispatcher()->OnSystemEvent(ESYSTEM_EVENT_CHANGE_FOCUS, 1, 0);
```

Comment:
Enable this again once we have our own `HardwareMouse`.

Reply:
Done in https://github.com/crymp-net/client-server/commit/1572295585c31cfb54ba11598c2ab0c8236e45df